packages/runner/lib/sdk/sdk.ts

```diff
@@ -324,14 +324,8 @@ export class NangoSyncRunner extends NangoSyncBase {
                     this.stringify()
                 );
 
-                if (response.status === 400) {
```

Comment:
the 400 is coming from there https://github.com/NangoHQ/nango/blob/master/packages/persist/lib/server.ts#L42-L45
Not sure why it is handled specifically here and even less sure why the error message doesn't match with the error.
just removing the special case and displaying the error provided by persist

Reply:
I added that so customers could understand a bit better the validation error when they would send invalid records